content/articles/css-hacks-for-web-developers/index.md

```diff
@@ -0,0 +1,178 @@
+
+1. ### Hovering effect delays
+In CSS, hovering is the change of the appearance of the component interface when the mouse is placed on it.
+
+Apart from being beautiful to the users' eyes, the Hovering effect provides more information to that element and also draws more attention.
+
+Below is an example snippet of a hovering effect:
+```html
+<html>
+  <body>
+    <h4>My website</h4>
+    <p>This is my first website</p>
+    <style>
+      /*Styles without hovering effect*/
+      h4 {
+        font-size: 70px;
+        color: red;
+      }
+      p {
+        font-size: 60px;
+        color: blue;
+      }
+      /* Styles, with hovering effect */
+      h4:hover {
+        color: black;
+        transition: all 0.5s ease; /*An element for delaying the hovering effect. */*
+      }
+      p:hover {
+        color: yellow;
+        transition: all 2s ease;
+      }
+    </style>
+  </body>
+</html>
+```
+In the code snippet above, we have styled elements `h4` and `p` with different hovering effects. `h4` will turn to color black when the mouse is placed on it with a delay of 0.5s while `p` will change to color yellow with a longer delay of 2.0s.
+
+2. ### How to use CSS to position content in the center
+To center content in the screen using CSS, most of the developers have to scratch their heads. According to research, it's not that it's difficult to do that task, it's just that there are many ways of doing it which confuse developers on which to use.
+
+For instance you can use  `position: absolute` to place you content in the center as shown in the snippet below:
+```css
+h4{
+  position: absolute;
+  left: 50%;
+  top: 50%;
+}
+```
+In the above style snippet, we have placed the content of element `h4` in the center. We have placed it halfway down the container with `top: 50%` and `left: 50%`.
+
+3. ### Fitting images to the page in CSS. 
+Have you ever come to a close website with images spilling over the site.❓️ Or have you ever designed a web app and on hosting it, the images spill all over the visitor's page?😞 If this ever happened am sure it caused a break on your site's layout and visitors turned away.
+
+However, using this CSS `hack`, I can guarantee you that your images will always fit on your end users' screen despite the devices they are using.
+
+Let's have a look at the example snippet below:
+```css
+img{
+  width: 500px;
+  height: 500px;
+  object-fit: scale-down;
+ }
+```
+In the above snippet, the `object-fit attribute specifies how an `image should be scaled to fit within its container.
+
+4. ### CSS style for visited links
+These styles are very important, yet some developers decide to avoid them. This sometimes causes trouble to the end-users as far as the usability of the web app is concerned.
+
```

Comment:
This line is ambiguous. Please provide examples of usability troubles that can be caused